components/formats-bsd/src/loci/formats/out/TiffWriter.java

```diff
@@ -545,7 +552,17 @@ else if (retrieve.getPixelsBinDataCount(series) == 0) {
   @Override
   public int getTileSizeX() throws FormatException {
     if (tileSizeX == 0) {
-      tileSizeX = super.getTileSizeX();
+      MetadataOptions options = getMetadataOptions();
+      boolean defaultTileWriting = TILE_WRITING_DEFAULT;
+      if (options instanceof FormatOptions) {
+        defaultTileWriting = ((FormatOptions) options).getBoolean(DEFAULT_TILE_WRITING_KEY, TILE_WRITING_DEFAULT); 
```

Comment:
Im not sure I quite understand, the default tile writing is not the same as a user specified tile size. If the user has specified a tile size that will override any other logic and be used instead. The default is purely providing a default value and behaviour.